src/main/java/qna/domain/Question.java

```diff
@@ -84,8 +90,35 @@ public boolean isDeleted() {
         return deleted;
     }
 
-    public List<Answer> getAnswers() {
-        return answers;
+    public DeleteHistories delete(User loginUser) throws CannotDeleteException {
+
+        if (!isOwner(loginUser)) {
+            throw new CannotDeleteException("질문을 삭제할 권한이 없습니다.");
+        }
+
+        this.setDeleted(true);
```

Comment:
Question을 반환하는 setDeleted(true) 대신 요렇게 바꿔봐도 좋을거 같아요! 😄 
```suggestion
        this.deleted = true;
```

Reply:
이경우 객체 내에서 직접 값에 접근하여 변경해도 되겠군요. 🤔 👍